packages/kit/src/vite/index.js

```diff
@@ -192,8 +196,6 @@ function kit() {
 
 				const new_config = vite_client_config();
 
-				warn_overridden_config(config, new_config);
```

Comment:
we should continue to warn in build mode. I'd don't think it'd happen without this